[1m[31mERROR   [0m test_contra_Voucher:test_contra_Voucher.py:345 Failed to enter first Account Ledger: Message: 
Stacktrace:
	GetHandleVerifier [0x0x7ff7ca4ee415+77285]
	GetHandleVerifier [0x0x7ff7ca4ee470+77376]
	(No symbol) [0x0x7ff7ca2b9a6a]
	(No symbol) [0x0x7ff7ca310406]
	(No symbol) [0x0x7ff7ca3106bc]
	(No symbol) [0x0x7ff7ca363ac7]
	(No symbol) [0x0x7ff7ca33864f]
	(No symbol) [0x0x7ff7ca36087f]
	(No symbol) [0x0x7ff7ca3383e3]
	(No symbol) [0x0x7ff7ca301521]
	(No symbol) [0x0x7ff7ca3022b3]
	GetHandleVerifier [0x0x7ff7ca7d1efd+3107021]
	GetHandleVerifier [0x0x7ff7ca7cc29d+3083373]
	GetHandleVerifier [0x0x7ff7ca7ebedd+3213485]
	GetHandleVerifier [0x0x7ff7ca50884e+184862]
	GetHandleVerifier [0x0x7ff7ca51055f+216879]
	GetHandleVerifier [0x0x7ff7ca4f7084+113236]
	GetHandleVerifier [0x0x7ff7ca4f7239+113673]
	GetHandleVerifier [0x0x7ff7ca4de298+11368]
	BaseThreadInitThunk [0x0x7ffe85327374+20]
	RtlUserThreadStart [0x0x7ffe8617cc91+33]